frontend/src/components/MeetingRoom/LayoutButton/LayoutButton.tsx

```diff
@@ -6,23 +6,30 @@ import useSessionContext from '../../../hooks/useSessionContext';
 import ToolbarButton from '../ToolbarButton';
 import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
-type LayoutButtonProps = {
+export type LayoutButtonProps = {
   isScreenSharePresent: boolean;
+  isPinningPresent: boolean;
```

Comment:
Could we change this to `isPinnedPresent` to match across components? Otherwise we have `isPinningPresent` and `isPinnedPresent`.

Reply:
funny, I started off with `isPinnedPresent` but Kongpheng suggested `isPinningPresent` 😃 @v-kpheng, are you okay if I change this back to `isPinnedPresent` to match how we use it across other components? 🙏

Reply:
Or, maybe we could rename the `isPinnedPresent` in the other component to `isPinningPresent`? I'm not tied to either variable name

Reply:
I changed it to `isPinningPresent` in other components 🙏